project-p2-layouts-master/code/index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Jobsurfing.ch</title>
+    <link rel="stylesheet" href="./style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <!-- dont forget the meta tag for responsiveness -->
+  </head>
+  <body>
+    <header>
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      <img src="Bilder/Logo-beige.PNG" alt="Logo-Jobsurfing"> 
+      <nav> 
+        <a href="#">About</a>
+        <a href="#">Für Jobsurfer</a>
+        <a href="#">Für Unternehmer</a>
+        <a href="#">Partner</a>
+      </nav>
+    </header>
+
+    <section class="cover">
+      <div class="image">
+        <img src="Bilder/cover_PixxlTeufel_ixabay.jpg"> 
+      </div>
+      <div class="center">
+        <h2>Brauchst du mehr Klarheit, wie deine berufliche Zukunft aussehen könnte?</h2>
+        <h1>Sammle ganz unkompliziert neue Joberfahrungen mit Jobsurfing.ch</h1>
+      </div>
+  </section>
+
+    <main class="information">
+       <section class="description">
+        <div>
+          <p>Jobsurfing.ch -  Die Schnittstelle zwischen ... und Unternehmen mit ... .</p>
+        </div>
+      </section>
+
+      <div class="subtitle">
+        <h2>Welche Vorteile bietet Jobsurfing.ch?</h2>
+      </div>
+
+      <section class="offers">
+          <div class="jobsurfers">
+          <h2>Für Jobsurfer</h2>
+        </div>
+        <div class="companies">
+          <h2>Für Unternehmen</h2>
+        </div>
+      
+      </section>
+
+      <div class="subtitle">
+         <h2>Fühlst du dich bei einer der folgenden Fragen angesprochen?</h2>
+      </div>
+
+      <section class="speech-bubbles">
+        <div class="question-blue">
+          <img src="#" alt="" >
```

Comment:
- If you don't have an image to show, remove these image tags.
- Don't leave the `alt` tags empty! They are important for e.g. accessibility